src/web/routes/application/check/get.js

```diff
@@ -1,16 +1,97 @@
-const pageContent = {
-  title: 'Check',
-  heading: 'Check'
+const { YES } = require('../are-you-pregnant/constants')
+
+const pageContent = ({ translate, claim }) => ({
+  title: translate('check.title'),
+  heading: translate('check.heading'),
+  sendApplicationHeader: translate('check.sendApplicationHeader'),
+  sendApplicationText: translate('check.sendApplicationText'),
+  buttonText: translate('buttons:acceptAndSend'),
+  checkRowData: buildCheckRowData(translate, claim)
```

Comment:
Subjective suggestion:

I would maybe pass `checkRowData` directly to the template (in `render` call). This would avoid passing `claim` to `pageContent()` just to pass it to `buildCheckRowData()`.

The `checkRowData` data structure is also quite different to the translated strings so maybe makes sense to separate.

Reply:
Done.